includes/admin/class-insert-term.php

```diff
@@ -47,4 +98,21 @@ public function create_new_term() {
             }
         }
     }
+
+    /**
+     * Regiseter taxonomy
+     *
+     * @return void
+     */
+    public function register_taxonomy() {
```

Comment:
This is a good way of registering taxonomies, do that for post types as well. Removing action hooks from individual post type classes are a good way of making classes testable. Move this function to the main `bediq.php`, so there will be two functions there. `register_post_types` and `register_taxonomies` in `bediq.php`